jenkins/clouds/ibm/weekly/cost_over_usage/run_policies.py

```
@@ -2,11 +2,15 @@
 IBM_ACCOUNT_ID = os.environ['IBM_ACCOUNT_ID']
+ES_HOST = os.environ.get('ES_HOST', '')
+ES_PORT = os.environ.get('ES_PORT', '')
+ES_USER = os.environ.get('ES_USER', '')
+ES_PASSWORD = os.environ.get('ES_PASSWORD', '')
 TO_MAIL = ['natashba@redhat.com']
 CC_MAIL = ['yinsong@redhat.com', 'ebattat@redhat.com', 'pragchau@redhat.com']
 USAGE_REPORTS_AUTHTYPE = 'iam'
 MAXIMUM_THRESHOLD = 1000
 QUAY_CLOUD_GOVERNANCE_REPOSITORY = os.environ['QUAY_CLOUD_GOVERNANCE_REPOSITORY']
 
 os.system(
```

Ruff (0.15.20)

[error] Starting a process with a shell, possible injection detected

(S605)